tasks.py

```diff
@@ -228,7 +228,8 @@ def docstrings(ctx: Context, paths: list[str] | None = None):
 def marker_coverage(
     ctx: Context,
 ):
-    pytest_cmds = ["pytest", "--verify-marker-coverage-and-exit"]
+    # -W do not error on warnings
+    pytest_cmds = ["pytest", "--verify-marker-coverage-and-exit", "-W=once"]
```

Comment:
This change prevents the `marker-converage` static analysis check from erroring on a warning without changing anything about our dependencies.

Reply:
Filtering in pyproject.toml instead